src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -122,6 +122,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)RemoveUnusedMembers\CSharpRemoveUnusedMembersDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)RemoveUnusedParametersAndValues\CSharpRemoveUnusedParametersAndValuesDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseCollectionInitializer\CSharpUseCollectionInitializerDiagnosticAnalyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)UseImplicitlyTypedLambdaExpression\UseImplicitlyTypedLambdaExpressionDiagnosticAnalyzer.cs" />
```

Comment:
>UseImplicitlyTypedLambdaExpressionDiagnosticAnalyzer.cs [](http://example.com/codeflow?start=84&length=55)

Is this file included in the PR?

Reply:
no.  this was for anotehr branch.  removing.